.github/workflows/tests.sh

```diff
@@ -29,7 +29,6 @@ verdi daemon stop
 pytest --noconftest ${SYSTEM_TESTS}/test_test_manager.py
 pytest --noconftest ${SYSTEM_TESTS}/test_ipython_magics.py
 pytest --noconftest ${SYSTEM_TESTS}/test_profile_manager.py
-python ${SYSTEM_TESTS}/test_plugin_testcase.py  # uses custom unittest test runner
```

Comment:
This isn't technically a deprecation is it?
I'm personally not too bothered if you remove it, but what was the reasoning for doing so?

Reply:
If I understood correctly, the unittest test case manager has been deprecated and this was just testing it. So no more need for it right?

Reply:
It was calling the `TestRunner` and `PluginTestCase` from `aiida.manage.tests.unittest_classes` which included a deprecation warning for v2.0 saying it was going to be removed. I am pretty sure we only support `pytest` from now on

Reply:
yeh actually thats fine